File: dart-jaguar/lib/model/util_content_add_ipfs_body.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

part 'util_content_add_ipfs_body.jser.dart';

class UtilContentAddIpfsBody {
  
  final String coluuid;
  
  final String dir;
  
  final String filename;
  
  final List<String> peers;
  
  final String root;
  

  UtilContentAddIpfsBody(
    

{
     this.coluuid = null,  
     this.dir = null,  
     this.filename = null,  
     this.peers = const [],  
     this.root = null 
    
    }
  );

  @override
  String toString() {
    return 'UtilContentAddIpfsBody[coluuid=$coluuid, dir=$dir, filename=$filename, peers=$peers, root=$root, ]';
  }
}

@GenSerializer()
class UtilContentAddIpfsBodySerializer extends Serializer<UtilContentAddIpfsBody> with _$UtilContentAddIpfsBodySerializer {

}

File: scala-akka-http-server/src/main/scala/io/swagger/server/model/Util.ContentAddIpfsBody.scala
package io.swagger.server.model


/**
 * @param coluuid 
 * @param dir 
 * @param filename 
 * @param peers 
 * @param root 
 */
case class Util.ContentAddIpfsBody (
  coluuid: Option[String],
  dir: Option[String],
  filename: Option[String],
  peers: Option[List[String]],
  root: Option[String]
)

